aspnet/security/cors/sample/src/CorsExample2/Startup.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.AspNet.Builder;
```

Comment:
This CORS doc is in the ASP.NET repo, so the content here should be generic to ASP.NET and not specific to MVC. The MVC CORS support is documented here: http://docs.asp.net/projects/mvc/en/latest/security/cors-policy.html